bazel/envoy_examples.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/single-page-app/verify.sh b/single-page-app/verify.sh
```

Comment:
This patch is added as a workaround for https://github.com/envoyproxy/examples/pull/384 . It can be removed after https://github.com/envoyproxy/examples/pull/384 is merged.